package.json

```diff
@@ -35,6 +35,12 @@
       ]
     },
     "commands": [
+      {
+        "command": "onevscode.show-compile-webview",
+        "title": "Compile",
+        "category": "ONE",
+        "_comment": "Call this when compiling compilable model(e.g., *.tflite, *.onnx, etc.)"
```

Comment:
Note: `_comment` was not defined by vscode nor nodejs. I used this non-standard key name to add comment. To distinguish this with standard key names, I put underbar.

Reply:
Is there any (doc) file that describes this? if not, how about adding this style to some doc file?

Reply:
(please merge this without my approval... left just a comment not a review opinion)

Reply:
Good idea. I will think about this and create a new issue.

Reply:
#420 was created.